README.md

```diff
@@ -14,9 +14,9 @@ This repo is a template for all EDGI templates. Replace this paragraph with a de
 | ---- | ----------- | ---------- |
 | [Environmental-Enforcement-Watch](https://github.com/edgi-govdata-archiving/Environmental-Enforcement-Watch) | **(This Repo!)** Project-wide documentation and issue tracking. | Markdown |
 | [EEW_Planning](https://github.com/edgi-govdata-archiving/EEW_Planning)| Project board and issue tracking for the coordination and planning of EEW events. | Markdown |
-| [ECHO-Hazardous-Waste](https://github.com/edgi-govdata-archiving/ECHO-Hazardous-Waste) | Jupyter notebooks for ECHO RCRA analysis to use with the EEW events.| Python, , Jupyter Notebook |
-| [ECHO-CWA-Compliance](https://github.com/edgi-govdata-archiving/ECHO-CWA-Compliance) | Use EPA's ECHO datasets to view compliance violations by quarter for state/congressional districts. | Python, Jupyter Notebook |
-| [ECHO-Air-Violations](https://github.com/edgi-govdata-archiving/ECHO-Air-Violations) | Jupyter notebook for examining ICIS-AIR_VIOLATION_HISTORY.| Python, Jupyter Notebook |
+| [ECHO-Hazardous-Waste](https://github.com/edgi-govdata-archiving/ECHO-Hazardous-Waste) | Jupyter notebooks for ECHO RCRA analysis to use with the EEW events.| Python, Jupyter Notebook |
+| [ECHO-Clean-Water-Act](https://github.com/edgi-govdata-archiving/ECHO-CWA-Compliance) | Use EPA's ECHO datasets to view compliance violations by quarter for state/congressional districts. | Python, Jupyter Notebook |
+| [ECHO-Clean-Air-Act](https://github.com/edgi-govdata-archiving/ECHO-Air-Violations) | Jupyter notebook for examining ICIS-AIR_VIOLATION_HISTORY.| Python, Jupyter Notebook |
 | [ECHO-COVID19](https://github.com/edgi-govdata-archiving/ECHO-COVID19) | Jupyter otebook for examining the effects of EPA's policy of not enforcing environmental protection rules during the pandemic.| Python, Jupyter Notebook |
```

Comment:
Can we fix the typo on this line. Should be "Jupyter notebook"

Not sure I can edit it myself since I don't have write permission on this repo.

Reply:
fixed!